client/apps/city_issues/views.py

```diff
@@ -6,53 +6,73 @@
 from datetime import date, datetime, time
 
 from django.views.generic import CreateView
-from django.views.generic.base import TemplateView
+from django.views.generic.base import TemplateView, View
 from django.contrib import messages
 from django.core import serializers
 from django.http import JsonResponse
 from django.shortcuts import redirect, render
 from django.urls import reverse
 from django.utils.timezone import make_aware
 
-from city_issues.models import Attachments, Issues
-from city_issues.forms.forms import EditIssue, IssueFilter, IssueForm
+from city_issues.models import Attachments, Issues, IssueHistory, User
+from city_issues.forms.forms import EditIssue, IssueForm
 
 
 class HomePageView(TemplateView):
     """Home page"""
     template_name = "home_page.html"
 
 
+class UserProfileView(View):
+
+    def get(self, request, user_id):
+        user = User.objects.get(id=user_id)
+        user_issues = IssueHistory.objects.filter(user=user).select_related('issue')
```

Comment:
Why do you select issue through `IssueHistory`, I think it return wrong data and issue duplicated.
I tried that way `user_issues = Issues.objects.filter(user=user).select_related('user')`

Reply:
You're right, I confused the models